code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -62,31 +84,34 @@
 		to_chat(user, "<span class='warning'>You're pretty sure you know exactly what this is used for and you can't seem to touch it.</span>")
 		return
 	if(!anchored)
-		to_chat(user, "<span class='cult italic'>You need to anchor [src] to the floor with a tome first.</span>")
+		to_chat(user, "<span class='cultitalic'>You need to anchor [src] to the floor with your dagger first.</span>")
 		return
 	if(cooldowntime > world.time)
 		to_chat(user, "<span class='cult italic'>The magic in [src] is weak, it will be ready to use again in [DisplayTimeText(cooldowntime - world.time)].</span>")
 		return
-	var/choice = alert(user,"You study the schematics etched into the forge...",,"Eldritch Whetstone","Zealot's Blindfold","Flask of Unholy Water")
-	var/pickedtype
+	var/choice = alert(user,"You study the schematics etched into the altar...",,"Eldritch Whetstone","Construct Shells","Flask of Unholy Water")
+	var/list/pickedtype = list()
 	switch(choice)
 		if("Eldritch Whetstone")
-			pickedtype = /obj/item/sharpener/cult
-		if("Zealot's Blindfold")
-			pickedtype = /obj/item/clothing/glasses/night/cultblind
+			pickedtype += /obj/item/sharpener/cult
+		if("Construct Shells")
+			pickedtype += /obj/structure/constructshell
+			pickedtype += /obj/structure/constructshell
 		if("Flask of Unholy Water")
-			pickedtype = /obj/item/reagent_containers/food/drinks/bottle/unholywater
+			pickedtype += /obj/item/reagent_containers/glass/beaker/unholywater
 	if(src && !QDELETED(src) && anchored && pickedtype && Adjacent(user) && !user.incapacitated() && iscultist(user) && cooldowntime <= world.time)
 		cooldowntime = world.time + 2400
+		for(var/N in pickedtype)
+			new N(get_turf(src))
+			to_chat(user, "<span class='cultitalic'>You kneel before the altar and your faith is rewarded with the [choice]!</span>")
 		var/obj/item/N = new pickedtype(get_turf(src))
```

Comment:
https://tgstation13.org/parsed-logs/basil/data/logs/2018/01/25/

The following runtime has occurred 9 time(s).
runtime error: Cannot create objects of type /list.
proc name: attack hand (/obj/structure/destructible/cult/talisman/attack_hand)
  source file: cult_structures.dm,107
  usr: Dante Smith (/mob/living/carbon/human)
  src: the altar (/obj/structure/destructible/cult/talisman)

Reply:
Didn't you spawn the item @ 105 anyway?